crates/shared/src/price_estimation/http.rs

```diff
@@ -135,6 +135,7 @@ impl HttpPriceEstimator {
                 AmmParameters::ConstantProduct(params) => tokens.extend(params.reserves.keys()),
                 AmmParameters::WeightedProduct(params) => tokens.extend(params.reserves.keys()),
                 AmmParameters::Stable(params) => tokens.extend(params.reserves.keys()),
+                AmmParameters::Concentrated(_) => (), // not used since no output amounts are calculated in driver
```

Comment:
Q: Is it reasonable to skip UniswapV3 liquidity in HttpPriceEstimator?

Reply:
IMO unless it's easy/clean to implement I think skipping univ3 is fine for now. At some point we should make sure price estimation uses the same liquidity as the final solving of an auction but I believe that would be quite a bit of work (this might come as a byproduct of the solver + driver co-location).

Reply:
I think its fine to exclude for now. We should create an issue for it though. Also, doesn't the HTTP solver return price estimates for this liquidity? Shouldn't just sending it to the HTTP solvers already make it get included in the price estimates (i.e. we don't need the ability to compute in/out amounts in code here).

Reply:
Yes, you are right, thanks. Not sure why I was convinced I need amounts for this. Anyway, added.